docs/topics/jvm/jvm-spring-boot-using-crudrepository.md

```diff
@@ -0,0 +1,97 @@
+[//]: # (title: Use Spring Data CrudRepository for database access)
+[//]: # (description: Work with Spring Data interface in a Spring Boot project written in Kotlin.)
+
+<microformat>
+    <p>This is the final part of the <strong>Getting started with Spring Boot and Kotlin</strong> tutorial. Before proceeding, make sure you've completed previous steps:</p><br/>
+    <p><img src="icon-1-done.svg" width="20" alt="First step"/> <a href="jvm-create-project-with-spring-boot.md">Create a Spring Boot project with Kotlin</a><br/><img src="icon-2-done.svg" width="20" alt="Second step"/> <a href="jvm-spring-boot-add-data-class.md">Add a data class to the Spring Boot project</a><br/><img src="icon-3-done.svg" width="20" alt="Third step"/> <a href="jvm-spring-boot-add-db-support.md">Add database support for Spring Boot project</a><br/><img src="icon-4.svg" width="20" alt="Fourth step"/> <strong>Use Spring Data CrudRepository for database access</strong></p>
+</microformat>
+
+In this part, you will migrate the service layer to use the [Spring Data](https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/repository/CrudRepository.html) `CrudRepository` instead of `JdbcTemplate` for database access.
+_CrudRepository_ is a Spring Data interface for generic [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) operations on a repository of a specific type.
+It provides several methods out of the box for interacting with a database.
+
+## Update your application
+
+First, you need to adjust the `Message` class for work with the `CrudRepository` API:
+
+1. Add the `@Table` annotation to the `Message` class to declare mapping to a database table.  
+   Add the `@Id` annotation before the `id` field.
+
+    > These annotations also require additional imports.
+    >  
+    {type="note"}
+
```

Comment:
Probably need to add one more line here because this note and code are visually merged on the staging.

<img width="778" alt="Screenshot 2022-09-09 at 21 36 19" src="https://user-images.githubusercontent.com/78360457/189430246-97f73e4d-b4b6-460d-b2c7-b51199c586c0.png">